controllers/groupsController.js

```diff
@@ -20,4 +22,64 @@ const list = async (req, res) => {
   }
 };
 
-module.exports = { list };
+const summary = async (req, res) => {
+  if (!isValidString(req.params.uid) || isEmptyString(req.params.uid)) {
+    return res.status(400).send({ message: "[InvalidUid] Error occured" });
+  }
+
+  try {
+    const postUpdateNewest = [];
+    const { uid } = req.params;
+    const keywordListResult = await keywordModel.find({ ownerUid: uid }).exec();
+    const keywordIdList = keywordListResult.map((keyword) => keyword._id);
+    const postCreatedLast = [];
+
+    for await (const id of keywordIdList) {
+      const postResult = await postModel.find({ keywordId: id }).sort({ createdAt: -1 }).exec();
+      postCreatedLast.push({
+        keywordId: postResult[0].keywordId,
+        createdAt: postResult[0].createdAt,
+      });
+    }
+
+    const postCreatedNewest = postCreatedLast?.reduce((prev, curr) => {
+      return new Date(prev.createdAt) <= new Date(curr.createdAt) ? curr : prev;
+    });
+    const dateNewest = postCreatedNewest.createdAt.toString();
+    const dateNewestStart = new Date(dateNewest).setHours(0, 0, 0, 0);
+    const dateNewestEnd = new Date(dateNewest).setHours(23, 59, 59, 999);
+    const groupListResult = await groupModel
+      .find({ ownerUid: uid })
+      .populate("keywordIdList", "keyword")
+      .exec();
+    const groupUpdatedNewest = groupListResult.filter((group) => {
+      return group.keywordIdList.some(
+        (keyword) => keyword._id.toString() === postCreatedNewest.keywordId.toString()
+      );
+    })[0];
+    const keywordList = groupUpdatedNewest.keywordIdList;
+
+    for await (const keyword of keywordList) {
+      const post = await postModel
+        .find({
+          keywordId: keyword._id,
+          createdAt: { $gte: dateNewestStart, $lte: dateNewestEnd },
+        })
+        .exec();
+
+      postUpdateNewest.push({ name: keyword.keyword, count: post.length });
```

Comment:
```suggestion
      if (postResult.length !== 0) {
        postUpdateNewest.push({ name: keyword.keyword, count: postResult.length });
      }
```
위 39-42 라인 이슈와 동일한 사유로, 쿼리 결과가 존재할 때에만 `postUpdateNewest` 최근 업데이트 게시물 데이터로 처리하기 위해 수정했습니다.

Reply:
```suggestion
      postUpdateNewest.push({ id: keyword._id, name: keyword.keyword, count: postResult.length });
```
클라이언트에서 리스트 렌더링을 위한 key 값으로 keywordId 값을 사용하고자 해서 수정하였습니다. 수정사항들이 여러 있는데 PR 전 미리 검토, 반영할 수 있게 하겠습니다!